### PR TITLE
Fix #72: List of all installed Phars

### LINKED ISSUE #72: List of all installed Phars
I would like to see not only a list of Phars installed in the current directory, but also a global list of what is installed where.

### PR DESCRIPTION
**Pull request by MacFJA**
Add an option (`--all`) to status command to list all installed phars.

----

```
$ php phive status
Phive 0.13.2-12-gff87560 - Copyright (C) 2015-2020 by Arne Blankerts, Sebastian Heuer and Contributors
PHARs configured in phive.xml:

Alias/URL       Version Constraint    Installed    Location                Key Ids             

phpab           ^1.25.1               1.25.6       ./tools/phpab           2A8299CE842DD38C    
phpunit         ^8.1                  8.3.4        ./tools/phpunit         4AA394086372C20A    
phpstan         ^0.11.5               0.11.12      ./tools/phpstan         8E730BA25823D8B5    
psalm           ^3.4.10               3.4.11       ./tools/psalm           8A03EA3B385DBAA1    
php-cs-fixer    ^2.15.3               2.15.3       ./tools/php-cs-fixer    E82B2FB314E9906E    
```

```
$ php phive status --all
Phive 0.13.2-12-gff87560 - Copyright (C) 2015-2020 by Arne Blankerts, Sebastian Heuer and Contributors
PHARs configured in your system:

Alias/URL       Version    Usage Location                              Key Ids             

dbunit          4.0.0      -                                           4AA394086372C20A    
phan            2.7.0      /tmp/plugin/vendor/bin/phan                 AFA6EAAB339B841E    
phan            2.7.0      /tmp/plugin_copy/vendor/bin/phan            AFA6EAAB339B841E    
php-cs-fixer    2.15.3     /home/me/github/phive/tools/php-cs-fixer    E82B2FB314E9906E    
phpab           1.25.6     /home/me/github/phive/tools/phpab           2A8299CE842DD38C    
phpcs           3.5.4      -                                           31C7E470E2138192    
phpstan         0.11.12    /home/me/github/phive/tools/phpstan         8E730BA25823D8B5    
phpunit         8.3.4      /home/me/github/phive/tools/phpunit         4AA394086372C20A    
phpunit         9.1.1      /tmp/plugin/vendor/bin/phpunit              4AA394086372C20A    
psalm           3.4.11     /home/me/github/phive/tools/psalm           8A03EA3B385DBAA1  
```

### CODE REVIEW

File: src/Factory.php [hunk 1]
```diff
@@ -144,7 +144,11 @@ public function getComposerCommand(): ComposerCommand {
 
     public function getStatusCommand(): StatusCommand {
         return new StatusCommand(
-            $this->getPhiveXmlConfig($this->request->getOptions()->hasOption('global')),
```
Comment (theseer) [inline src/Factory.php:147]: Is there a reason you removed the global option and hard-coded it?

File: src/Factory.php [hunk 2]
```diff
@@ -144,7 +144,11 @@ public function getComposerCommand(): ComposerCommand {
...
+            new StatusCommandConfig(
+                $this->request->parse(new StatusContext()),
+                $this->getPhiveXmlConfig(false),
```
Comment (theseer) [inline src/Factory.php:149]: This used to be optional via `global` option.

**Comment by theseer** [comment]
And we should probably make sure to update the [help text](https://github.com/phar-io/phive/blob/master/src/commands/help/help.md) in the help command.

File: src/Factory.php (hunk 1 above)
Comment (MacFJA) [reply src/Factory.php:147]: Nope!
My bad, it's a leftover of a previous test. I'll correct it.

**Comment by theseer** [comment]
Thanks! :+1:

**Comment by theseer** [comment]
I tweaked the implementation a little and will likely craft a release tomorrow.

**Pull request merged**